scripts/buildkite/tpp-llvm.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  NPROCS_LIMIT_LINK: "8"
+
+steps:
+  - label: "TPP-LLVM"
+    concurrency: 1
+    concurrency_group: "tpp-llvm"
+    command: "scripts/buildkite/check_llvm.sh || \
+              ${SRUN} --partition=spr --time=1:00:00 -- \
```

Comment:
Do we give back the partition to slurm or we keep it for 1h? I think 30 minutes are enough.

Reply:
It is returned when the job is done. The 1h is just the upper limit.
I kept the same higher time allocation as it was in the old tpp-llvm setup.

Reply:
I'll lower it to 30min to be consistent with others, should be enough.